src/PositionService.Mappers/Db/DbPositionUserMapper.cs

```diff
@@ -19,11 +29,28 @@ public DbPositionUser Map(ICreateUserPositionRequest request)
         Id = Guid.NewGuid(),
         UserId = request.UserId,
         PositionId = request.PositionId,
-        Rate = request.Rate,
         IsActive = true,
         CreatedBy = request.CreatedBy,
         CreatedAtUtc = DateTime.UtcNow
       };
     }
+
+    public DbPositionUser Map(CreatePositionUserRequest request)
+    {
+      if (request == null)
```

Comment:
Why do we write in some places "== null" in others "is null"?

Reply:
fixed